tests/Doctrine/Tests/ORM/Query/LanguageRecognitionTest.php

```diff
@@ -150,6 +150,9 @@ public function invalidDQL(): array
             ['SELECT \'foo\' AS foo: FROM Doctrine\Tests\Models\CMS\CmsUser u'],
             ['SELECT \'foo\' AS foo:bar FROM Doctrine\Tests\Models\CMS\CmsUser u'],
 
+            /* Checks for incomplete queries */
+            ['SELECT \'foo\' FROM Doctrine\Tests\Models\CMS\CmsUser u ORDER BY '],
```

Comment:
Is this test case looks good ? 

cc @greg0ire @derrabus

Reply:
The `\'foo\'` part is a bit distracting. I would have used
```suggestion
            'incomplete query' => ['SELECT u FROM Doctrine\Tests\Models\CMS\CmsUser u ORDER BY '],
```